ts/Extensions/Boost/WGLRenderer.ts

```diff
@@ -748,11 +748,11 @@ function GLRenderer(
                 continue;
             }
 
-            if (nx && nx >= xMin && nx <= xMax) {
+            if (nx && nx >= xMin && nx <= xMax && series.type !== 'scatter') {
```

Comment:
Hard checking for `series.type` should not be used because it will fail for inherited children of scatter. Instead, use `!series.is('scatter')`, which will also be true for inherited children. Alternatively, in this case we may actually be looking for `&& series.is('line')`

Reply:
But scatter inherits from line right? I'll do `!series.is('scatter')`